lib/Github/HttpClient/CachedHttpClient.php

```diff
@@ -105,4 +114,31 @@ public function getLastResponse($force = false)
 
         return ($force) ? $lastResponse : $this->lastCachedResponse;
     }
+
+    /**
+     * Format the path and add query parameters if they exist.
+     *
+     * @param string $path
+     * @param array  $options
+     * @return void
+     */
+    private function formatPath($path, array $options)
+    {
+        $this->path = $path;
+
+        if (array_key_exists('query', $options) && !empty($options['query'])) {
+            $this->path .= '?';
+
+            $i = 0;
+            foreach ($options['query'] as $key => $value) {
```

Comment:
use ``http_build_query($options['query'], '', '&')`` instead of reimplementing it in an incomplete way